### ISSUE TITLE
Examples for deletion of mandate and users

### ISSUE THREAD

**Issue by Thorazine** [mollie/laravel-mollie#106]
Mollie v2 is working fine with sepa and other payment systems. But I need a way for the user (me) to invalidate or, better yet, delete mandates.

Also, I would like to know how to delete created users from mollie's database. I know I can do so from the panel, but an API example would be better.

**Comment by sandervanhooft** [comment]
Hi @thorazine,

Should be possible. What have you tried so far?

**Comment by Thorazine** [comment]
Well, in the examples I found how to retrieve mandates, and that works fine. So I have the mandate id's as well as the user id's. But looking through the package and docs I could not find a delete function, which I tried in the code (fail). I would have suspected something like:

```
Mollie::api()->mandates()->listFor($this->customer)->delete();
```
Or
```
Mollie::api()->mandates()->delete($this->mandate_id)
```
But no joy. I do know it's possible seeing the api docs say it can be done. But doing it with this package riddles me.
